salt/modules/network.py

```diff
@@ -1822,6 +1832,7 @@ def default_route(family=None):
     ret = []
     for route in _routes:
         if family:
+            def_route_family = default_route[family]
```

Comment:
This variable is unused as well.

Reply:
Gone through that area and ensured there should be no more droppings left from debugging the issue.
Sorry for that.